### PR TITLE
Change #text to be closer to what a browser displays

### PR DESCRIPTION
**Pull request by twalpole**
Playing with potential changes for Issue #1353

### CODE REVIEW

**Comment by twalpole** [comment]
@abotalov Any comments/feedback on this as it relates to Issue #1353

File: lib/capybara/spec/session/ancestor_spec.rb [hunk 1]
```diff
@@ -17,7 +17,7 @@
 
   it "should find the ancestor element using the given locator and options" do
     el = @session.find(:css, '#child')
-    expect(el.ancestor('//div', text: 'Ancestor Ancestor Ancestor')[:id]).to eq('ancestor3')
+    expect(el.ancestor('//div', text: "Ancestor\nAncestor\nAncestor")[:id]).to eq('ancestor3')
```
Comment (peetar) [inline lib/capybara/spec/session/ancestor_spec.rb]: After upgrading our gems, we ended up with a few hundred test failures requiring an update like this. (Yes, we foolishly assumed the whitespace normalization would never change and used this pattern all over our test suite). Anybody have a suggestion on how we might solve this without having to manually update a whole bunch of assertions?

Comment (twalpole) [reply lib/capybara/spec/session/ancestor_spec.rb]: @peetar You can use `Capybara.default_normalize_ws = true` - although that will most likely go away in Capybara 4 - so you're going to want to update your tests at some point.

Comment (peetar) [reply lib/capybara/spec/session/ancestor_spec.rb]: Thanks for the suggestion, might use it as a temporary workaround

**Pull request merged**